core/src/main/java/com/linecorp/armeria/common/MediaTypeSet.java

```diff
@@ -60,16 +60,36 @@
 
     private final MediaType[] mediaTypes;
 
+    /**
+     * Returns the {@link MediaTypeSet} which is composed of the specified {@link MediaType}s.
+     */
+    public static MediaTypeSet of(MediaType... mediaTypes) {
+        return of(ImmutableList.copyOf(requireNonNull(mediaTypes, "mediaTypes")));
```

Comment:
Better calling the constructor directly after `copyOf()`, given `of()` calls `copyOf()` again?

Reply:
Yes, we should.